client/src/components/common/Input/index.js

```diff
@@ -26,4 +28,28 @@ export default ({
       />
     </label>
   );
+}
+
+Input.propTypes = {
+  label: PropTypes.string,
+  id: PropTypes.number,
+  type: PropTypes.string,
+  placeholder: PropTypes.string,
+  className: PropTypes.string,
+  htmlFor: PropTypes.string,
+  value: PropTypes.string,
+  onChange: PropTypes.func,
+  name: PropTypes.string,
+};
+
+Input.defaultProps = {
+  label: '',
+  id: 0,
+  type: 'text',
+  placeholder: '',
+  className: 'input-type',
```

Comment:
same as above, the input already has class input-type, pls set it to empty string

Reply:
Done ^_^